packages/webcomponents/src/components/business-forms/owner-form/identity-address/identity-address-form.tsx

```diff
@@ -22,6 +24,10 @@ export class BusinessAddressForm {
     this.address = { ...this.address };
   }
 
+  constructor() {
+    this.inputHandler = this.inputHandler.bind(this);
```

Comment:
We can set `inputHandle` as an arrow function to get `this` bound automatically.

e.g
```
inputHandler = (name: string, value: string) => {
...
}
```

Reply:
Good call - just updated that one! Thank you!